perun.go

```diff
@@ -168,10 +168,10 @@ type WireBus interface {
 //
 // It defines methods for deploying contracts; validating deployed contracts and instantiating a funder, adjudicator.
 type ChainBackend interface {
-	DeployAdjudicator() (adjAddr pwallet.Address, _ error)
-	DeployAsset(adjAddr pwallet.Address) (assetAddr pwallet.Address, _ error)
+	DeployAdjudicator(onChainAddr pwallet.Address) (adjAddr pwallet.Address, _ error)
+	DeployAsset(adjAddr, cred pwallet.Address) (assetAddr pwallet.Address, _ error)
 	ValidateContracts(adjAddr, assetAddr pwallet.Address) error
-	NewFunder(assetAddr pwallet.Address) pchannel.Funder
+	NewFunder(assetAddr, addr pwallet.Address) pchannel.Funder
```

Comment:
what addr is the second argument, receiverAddr ?

Reply:
It is onChainAddr, I have updated the same in this [commit](https://github.com/hyperledger-labs/perun-node/pull/114/commits/aebcf94494c5d13ad272ecadd28edfe7b22dc659).